src/content/docs/accounts/accounts-billing/account-setup/student-program.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: New Relic for Students 
+metaDescription: Sign up for a New Relic student account 
+freshnessValidatedDate: never
+---
+
+Are you a student who is trying to learn about gathering and analyzing hardware and software telemetry? You can do this with New Relic for Students. This is a program that offers you free access to data, users, and New Relic features&mdash;even more options than you'd get from the Free Edition. 
+
+## Benefits [#student-benefits]
+
+Eligible students will receive access to a free New Relic for Students account for two years, which includes:
```

Comment:
Drop the "will"

Reply:
"free New Relic for Students account" is quite a mouth full. Maybe there's a simpler way to rephrase this?

Reply:
@paperclypse how about this: "Eligible students receive free access to a New Relic for Students account for two years, which includes:..."

Reply:
That seems better